src/SFML/Audio/InputSoundFile.cpp

```diff
@@ -106,7 +123,10 @@ std::optional<InputSoundFile> InputSoundFile::openFromMemory(const void* data, s
     // Pass the stream to the reader
     auto info = reader->open(*memory);
     if (!info)
+    {
+        err() << "Failed to open input sound file from memory (reader open failure)\n" << std::endl;
```

Comment:
In other places (e.g. in the new audio impl.), we only specify the actual error that happens without prefacing in what context this happened, i.e. "Failed to open reader"

Reply:
I think the additional context provided here can be useful in tracking down an eventual issue or bug. Can you point me to the place you're referring to? I've checked some Miniaudio-related `err()` calls and they all seem to provide quite a bit of context/information.

Reply:
Maybe it was something else, can't remember where I had the discussion.
Looking at the other examples, it might be more in line to do:

```
Failed to open input sound file from memory: reader open failure
```

Also there shouldn't be both a `\n` and `std::endl`

Reply:
> Looking at the other examples, it might be more in line to do:

Most of the examples I've found use parentheses to add additional information/context to an error message. I've grepped for `err() <<`. The language we use is not very consistent, but I'd rather deal with the consistency issue in a separate PR -- resolving for now.